lib/commands/uninstall.js

```diff
@@ -35,8 +36,8 @@ class UninstallCommand extends Command {
             task: () => this.ui.sudo(`rm -rf ${path.join(instance.dir, 'content')}`)
         }, {
             title: 'Removing related configuration',
-            task: () => {
-                this.system.setEnvironment(!fs.existsSync(path.join(instance.dir, 'config.production.json')));
+            task: (_, context) => {
+                this.system.setEnvironment(context._environment);
```

Comment:
I'm not sure how much we need to do this store and fetch dance. Do you know why it was implemented like this previously?